index.html

```diff
@@ -0,0 +1,76 @@
+  <link rel="stylesheet" href="responsive-style.css">
+  <script src="script.js" defer></script>
+  <script src="mobileMenu.js" defer></script>
+</head>
+<body>
+  <section id="home">
+    <nav>
+      <a href="index.html"><img src="img/logo.png" alt="" class="cclogo"></a>
+      <img src="img/mobile_menu.png" alt="" class="hamburger">
+      <ul class="link-items">
+        <li><a href="about.html">About</a> <img src="img/close_mobile_menu.png" alt=""></li>
+        <li><a href="index.html">Program</a></li>
+        <li><a href="index.html">Join</a></li>
+        <li><a href="index.html">Sponsor</a></li>
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html">CC Campaign</a></li>
+      </ul>
+    </nav>
+    <article class="welcome">
+      <p class="hello">"Hello! Sharing World"</p>
+      <h1 class="main-heading">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <P class="joyful">A joyful festival where people from over 80 countries who create positive change gather and share their thoughts will be held in Korea in October.</P>
+      <p class="schedule">2022.10.14 (Wed) ~ 17 (Sat)</p>
+      <p class="museum">@National Museum of Korea, Art Center Nabi, etc.</p>
+    </article>
+    <article class="main-program">
+      <h3>Main Program</h3>
+      <hr class="red-main-program">
+      <div class="programs">
+      <div class="program-activity"><img src="img/ponents_icon.png" alt="" class="program-icon"> <span class="program-title">Lecture</span><p class="program-detail">Listen to the speakers from various cooutries about the messeges of sharing and opeinging.</p></span></div>
+      <div class="program-activity"><img src="img/exhibition_icon.png" alt="" class="program-icon"> <span class="program-title">CC Exhibition</span><p class="program-detail">Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p></div>
+      <div class="program-activity"><img src="img/forum_icon.png" alt="" class="program-icon"> <span class="program-title">Forum</span><p class="program-detail">Have the time to share your thoughts and opinions with experts for each topic.</p></div>
+      <div class="program-activity"><img src="img/workshop_icon.png" alt="" class="program-icon"> <span class="program-title">Workshop</span><p class="program-detail">Try creating your own work using open source license rather than just watching.</p></div>
+      <div class="program-activity"><img src="img/cc-party.png" alt="" class="program-icon"> <span class="program-title">CC Ignite</span><p class="program-detail">Get opportunities to network with CC affiliate around the world, also after the summit</p></div>
+    </div>
+      <span class="seeWholeProgram"><p>SEE THE WHOLE PROGRAM</p></span>
+      <button class="joinCC">Join CC Global Summit 2015</button>
+      
+    </article>
+    <article class="speakers">
+      <h3 class="speaker-heading">Featured Speakers</h3>
+      <div class="speaker-group">
+      </div>
```

Comment:
- [ ] Please add the `line` under the FEATURE SPEAKERS text and the line under the description in red text
- [ ] The box on the top left of the speaker image is referring to the grilled image on the original template

<img width="1029" alt="3" src="https://user-images.githubusercontent.com/77843050/211072580-4fe008c6-8714-4827-93ea-a4d7e0dc46c9.png">